Assets.Scripts.Core.History/HistoryLine.cs

```diff
@@ -8,13 +10,36 @@ public class HistoryLine
 
 		public string TextJapanese;
 
-		public AudioInfo VoiceFile;
+		public int EnglishHeight { get; private set; }
 
-		public HistoryLine(string english, string japanese, AudioInfo voice)
+		public int JapaneseHeight { get; private set; }
+
+		public List<List<AudioInfo>> VoiceFiles;
+
+		public HistoryLine(string english, string japanese)
 		{
 			TextJapanese = japanese;
 			TextEnglish = english;
-			VoiceFile = voice;
+			VoiceFiles = new List<List<AudioInfo>>();
+		}
+
+		private int CalculateHeight(TextMeshPro measurer, TextMeshProFont font, string text)
```

Comment:
Maybe measurer isn't the best name in this case...it distracted me from seeing that it was just the TextMeshPro instance.  No big deal.

Reply:
I called it `measurer` because it's only used to measure the text in the environment it will be displayed in in the future, but not actually used for displaying anything

Maybe I should have called it `targetLocation` or something

Actually I probably should have stuck a documentation comment above it either way